config/initializers/recaptcha.rb

```diff
@@ -1,8 +1,8 @@
 # config/initializers/recaptcha.rb
-if Rails.env == "production"
+if Rails.env.production?
   Recaptcha.configure do |config|
-    config.site_key  = '6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI'
-    config.secret_key = '6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe'
+    config.site_key = ENV.fetch("RECAPTCHA_SITE_KEY")
```

Comment:
Oh, now that i remember, i think we wanted to keep the ` | '6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI'` because our test servers run in production and we like them to have the recaptcha option set. Do you think you could add that fallback back in? Sorry, i'd forgotten!

Reply:
@jywarren aaaah ok I understand! In that case I can't use fetch either, because it returns a key error and not a nil!